internals/get-package-json/package.json

```
@@ -0,0 +1,28 @@
+{
+  "private": true,
+  "name": "@vercel-internals/get-package-json",
+  "types": "dist/index.d.ts",
+  "main": "dist/index.js",
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "test": "pnpm test-unit",
```

```suggestion
    "test": "jest --env node --verbose --runInBand --bail"
```

All the other packages use this

---

oh i see i got it backwards